domain/BUILD.bazel

```diff
@@ -124,6 +124,9 @@ kt_android_library(
         "//utility/src/main/java/org/oppia/android/util/logging:event_logger",
         "//utility/src/main/java/org/oppia/android/util/logging:log_uploader",
         "//utility/src/main/java/org/oppia/android/util/networking:network_connection_util",
+        "//utility/src/main/java/org/oppia/android/util/parser/html:exploration_html_parser_entity_type",
+        "//utility/src/main/java/org/oppia/android/util/parser/image:glide_image_loader_module",
+        "//utility/src/main/java/org/oppia/android/util/parser/image:image_parsing_annonations",
```

Comment:
To finish this up, can you mention which classes actually need these dependencies (now that you've determined that these are the minimum needed)? I think, for example, that glide_image_loader_module is probably not used by domain and instead is needed by the app module, and maybe by tests. I just want to make sure that each of these dependencies make sense to be specifically *here* (versus in a package closer to the code that uses them).

Reply:
exploration_html_parser_entity_type is used in [PrimeTopicAssetsControllerImpl.kt](https://github.com/yash10019coder/oppia-android/blob/3a0afb4eb8222d4ff3acd2e2e660b85911d35229/domain/src/main/java/org/oppia/android/domain/topic/PrimeTopicAssetsControllerImpl.kt#L52)
image_parsing_annonations is used in [PrimeTopicAssetsControllerImpl.kt](https://github.com/yash10019coder/oppia-android/blob/3a0afb4eb8222d4ff3acd2e2e660b85911d35229/domain/src/main/java/org/oppia/android/domain/topic/PrimeTopicAssetsControllerImpl.kt#L56)
glide_image_loader_module wasn't used in domain it was used in app module so I added it as a dependency in app module

Reply:
That's great & helpful--thanks!